frontend/src/screens/ProductScreen.js

```diff
@@ -9,13 +9,16 @@ import Rating from '../components/Rating';
 export default function ProductScreen(props) {
   const dispatch = useDispatch();
   const productId = props.match.params.id;
+  const [qty, setQty] = useState(1);
   const productDetails = useSelector((state) => state.productDetails);
   const { loading, error, product } = productDetails;
 
   useEffect(() => {
     dispatch(detailsProduct(productId));
   }, [dispatch, productId]);
-
+  const addToCartHandler = () => {
+    props.history.push(`/cart/${productId}?qty=${qty}`);
```

Comment:
const navigate=useNavigate();
navigate(`/cart/${productId}?qty=${qty}`);

Reply:
navigate(`/cart/${id}?qty=${qty}`)

missing the backticks